recipes/connected-components-3d/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "connected-components-3d" %}
+{% set version = "3.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/connected-components-3d-{{ version }}.tar.gz
+  sha256: 7bc93617e1bccba45bae57f8b315a1fa247ab627dc86c27d27a0aea2babce680
+
+build:
+  skip: true  # [py>=38 or py2k]
```

Comment:
py37 only? that's weird :( Their setup.py shows [otherwise](https://github.com/seung-lab/connected-components-3d/blob/master/setup.py#L57-L62).

Reply:
Yes https://github.com/seung-lab/connected-components-3d/blob/master/setup.py#L29 

`python_requires="~=3.7", # >= 3.7 < 4.0` 

Maybe we can remove it?

Reply:
Yep I think so. I'll add the change.